enterprise_gateway/services/processproxies/container.py

```diff
@@ -230,8 +237,13 @@ def get_initial_states(self):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def get_container_status(self, iteration_string):
-        """Return current container state."""
+    def get_error_states(self):
+        """Return list of states indicating container is starting (includes running)."""
```

Comment:
Wrong docstring.  Let's convey its lowercase as well.
```suggestion
        """Returns the list of error states (in lowercase)."""
```